### ISSUE TITLE
umi masked as N when using CallDuplexConsensusReads

### ISSUE THREAD

**Issue by skywyx** [fulcrumgenomics/fgbio#522]
Hi fgbio team:
   Thank for your wonderful job.
   CallDuplexConsensusReads masks some umis in RX tags with 'N'
   Is that designed to be ? why?
   looking forward for your reply
`java -jar fgbio-0.8.1.jar CallDulplexConcensusReads -i test.umi.group.bam -o test.dulplex.bam`
[test.zip](https://github.com/fulcrumgenomics/fgbio/files/3532255/test.zip)

**Comment by nh13** [comment]
@skywyx my apologies for the late reply, but we also build a consensus call for the bases in the `RX` tags across the input reads for that duplex, so we may have some some disagreement across `RX` tags that cause ambiguity (i.e. they are equally likely) so they would be output as `N`

**Comment by skywyx** [comment]
@nh13 I got your point. But I'm still confused. 
Did you see the reads I uploaded in [test.zip](https://github.com/fulcrumgenomics/fgbio/files/3532255/test.zip) ? The two pairs of reads has `RX` tags  `ACTGATG-TACGCTA` and `TACGCTA-ACTGATG` respectively. And they are grouped to single group by `GroupReadsByUmi`(i.e. they have common MI tags prefix) .  In theory, thier `RX` tags completely agree with each other. But the results is most bases in `RX` were output as `N`.
